htdocs/web_portal/controllers/admin/view_service_type.php

```diff
@@ -24,31 +24,32 @@
 
 function view_service_type()
 {
-     //Check the user has permission to see the page, will throw exception
-    //if correct permissions are lacking
-    checkUserIsAdmin();
+    $params = [];
+
+    $identifier = Get_User_Principle();
+    $user = \Factory::getUserService()->getUserByPrinciple($identifier);
+
+    /**
```

Comment:
We can take this oppurtunity to remove this comment, it doesn't add any additional context to the code IMO.